tests/nativeapp/codegen/snowpark/test_extension_function_utils.py

```diff
@@ -7,6 +8,68 @@
 from tests.testing_utils.files_and_dirs import temp_local_dir
 
 
+def test_sanitize_ex_fn_attribute():
```

Comment:
See my comment below about testing implementation vs contract. For example, say we want to split up this method per parameter type, we have to rewrite all tests.

Reply:
Made some changes to the two testing files. Closing comments that relate to this as an acknowledgement as they would be on outdated code state, please open new comments for feedback.

As of the commit after this reply, `test_extension_function_utils.py` cov stands at 99%, and `test_python_processor.py` is at 98%.